Package.swift

```diff
@@ -11,7 +11,6 @@ let package = Package(
     .library(name: "PactConsumerSwift", targets: ["PactConsumerSwift"])
   ],
   dependencies: [
-    .package(url: "https://github.com/Quick/Quick.git", from: "2.1.0"),
```

Comment:
Unfortunately SwiftPM doesn't seem to be pulling this in automatically as `Quick/Nimble`'s dependency.

Reply:
Quick depends on Nimble, so it shouldn't be required.

Reply:
You're right (just went to check the deps).